src/fixtures/articles.js

```diff
@@ -0,0 +1,1631 @@
+export default [
+  {
+    id: "vG6Uo8cQ30eeEh9",
+    person: {
+      profileId: "Tkq1eLp2cFFNf0u",
+      firstName: "Noemie",
+      lastName: "Beier",
+      img: ""
+    },
+    createAt: "2019-06-09T15:03:23.000Z",
+    title: "To Love and to Be Loved - This Is the Psychology of Love",
+    typeArticle: ["featured news", "trend"],
+    shortDescription:
+      "September 2019 - Whether we are aware of it or not, love is all around us. If only we could realize it. There are many people who love us. Think of your family, friends, and the partners who cherish us. But the biggest question to ask ourselves is what love is.",
+    slider: "",
+    quoute:
+      "However, the psychology of love looks at things in a deeper way.\n Love changes how people view others and other things.\n How we do it matters to others and also ourselves.",
+    longDescription:
```

Comment:
longDescription is array 
```
{longDescription: [
"September 2019 - Whether we are aware of it or not, love is all around us. If only we could realize it. There are many people who love us. Think of your family, friends, and the partners who cherish us. But the biggest question to ask ourselves is what love is.", 
"Different people interpret it in a variety of ways. Some take it to be the emotional feeling they share with their partners (for those who are in a relationship or marriage). For others, it is that care that they receive from their parents, siblings, and other people who are close to them. But all of this brings us to one conclusion: love is everywhere, even if some people are oblivious of it",
 "However, the psychology of love looks at things in a deeper way. It can be said that different types of love do exist and all affect people in many ways. Do you know that love can change the way a person behaves? Yes, it can. This is one of the factors that we will look at in detail here in a bit." 
]}
```